minio/helpers.py

```diff
@@ -116,7 +116,7 @@ def dump_http(method, url, request_headers, response, output_stream):
                                                     http_path))
 
     for k, v in list(request_headers.items()):
-        if k is 'authorization':
+        if k == 'authorization':
             # Redact signature header value from trace logs.
             v = re.sub(r'Signature=([[0-9a-f]+)', 'Signature=*REDACTED*', v)
```

Comment:
It looks suspicious. Is not one of `[` redundant?

Also, `(` and `)` are not needed.
```suggestion
            v = re.sub(r'Signature=[0-9a-f]+', 'Signature=*REDACTED*', v)
```

Reply:
@serhiy-storchaka this PR doesn't need to address this. Feel free to send PR yourself.